### ISSUE TITLE
Missing Query argument comments when merged types

### ISSUE THREAD

**Issue by lkspc** [Urigo/merge-graphql-schemas#136]
In my schema definitions, I have commented the query argument: 
```graphql
type Query {
  users(
    # filter user by name
    name: String
    
    # limit for pagination
    limit: Int
  ): [User]
}
```

But after calling `mergeType`, the merged schema is:
```graphql
type Query {
  users(name: String, limit: Int): [User]
}
```

How could I resolve this problem?

**Comment by murielsilveira** [comment]
I could make it work with a different comment notation, try it like this:

```graphql
type Query {
  users(
    "filter user by name"
    name: String
    
    "limit for pagination"
    limit: Int
  ): [User]
}
```

I figure this out by checking these issues:

- https://github.com/apollographql/graphql-tag/issues/81
- https://github.com/apollographql/graphql-tag/issues/107

I hope it helps.

**Comment by lkspc** [comment]
Thanks for your solution, it works, though it's non-standard comments

**Comment by kamilkisiela** [comment]
Should be [fixed](https://github.com/Urigo/merge-graphql-schemas/pull/196/files#diff-ee59d0638e75272d8511d2bc3e6ae758R769) by https://github.com/Urigo/merge-graphql-schemas/pull/196

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`. Let us know if it works :)
